pxf/pxf-api/src/main/java/org/apache/hawq/pxf/api/FilterParser.java

```diff
@@ -63,7 +63,9 @@
         HDOP_EQ,
         HDOP_NE,
         HDOP_AND,
-        HDOP_LIKE
+        HDOP_LIKE,
```

Comment:
I don't think we need a separate enum for unary operator. We have separate enums currently based on how HAWQ bridge separates a/c/o operators. Also we will soon introduce IN and other operators as well.